php/libraries/User.class.inc

```diff
@@ -306,6 +306,19 @@ class User extends UserPermissions implements
         return $this->userInfo['ProjectIDs'];
     }
 
+    /**
+     * Get the user's projects
+     *
+     * @return Project[]
+     */
+    function getProjects(): array
+    {
+        return array_map(
+            fn($projectID) => \Project::getProjectFromID($projectID),
+            $this->userInfo['ProjectIDs']
```

Comment:
This should be `$this->getProjectIDs()`. It makes it easier for static analysis tools (and humans) to determine the type.